client.go

```diff
@@ -0,0 +1,63 @@
+package nomadclient
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	dphttp "github.com/ONSdigital/dp-net/http"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Nomad struct {
+	Client dphttp.Clienter
+	URL string
+	// Name needed for checker later.
+	// Name string
+}
+
+func NewClient(NomadEndpoint, NomadCACert string, NomadTLSSkipVerify bool) (*Nomad, error){
+
+	dpHTTPClient := *dphttp.DefaultClient
+
+	if strings.HasPrefix(NomadEndpoint, "https://") {
+		fmt.Print("in if https")
+		var tlsConfig *tls.Config
+		if NomadCACert != "" {
+			fmt.Print("in nomad CA if")
+			caCertPool, _ := x509.SystemCertPool()
```

Comment:
You need to handle the error that this can return.

Reply:
Question: Do you need to create a Cert Pool each time? Would this be one pool per client or is the pool shared between all clients that are created?